_data/education.yml

```diff
@@ -317,6 +317,15 @@ websites:
       facebook: royalroadsu
       img: royalroads.png
       tfa: No
+      
+    - name: Ryerson University
+      url: https://www.ryerson.ca
+      tfa: Yes
+      img: ryerson.png
+      hardware: Yes
+      software: Yes
+      phone: Yes
```

Comment:
I can't find anything on the documentation link you provided that talks about 2FA through phone calls. Are you sure they support it?

Reply:
Whoops my bad, I will go ahead and change it in a second.